kernel/ksu.c

```diff
@@ -53,10 +53,10 @@ int __init kernelsu_init(void)
 
 	ksu_uid_observer_init();
 
-#ifdef CONFIG_KPROBES
```

Comment:
Do we need to delete this?

Reply:
already checked in ksu_enable_ksud and ksu_enable_sucompat